lib/signature.c

```diff
@@ -199,7 +200,7 @@ rpmRC rpmGenerateSignature(char *SHA256, char *SHA1, uint8_t *MD5,
     if (spaceSize > 0) {
 	reservedSpace = (uint8_t *)xcalloc(spaceSize, sizeof(char));
 	rpmtdReset(&td);
-	td.tag = RPMSIGTAG_RESERVEDSPACE;
+	td.tag = reserveTag;
```

Comment:
Actually this series should probably just leave rpmGenerateSignature() alone. That only gets called by the build code, and we're not touching that here. Dropping it would also get rid of the highly confusing "Drop support for v6 RPMSIGTAG_RESERVED" commit - it doesn't drop support for the tag but just the part in here.